File: app/src/main/java/com/lc/bangumidemo/MyRetrofit/APIinterface/APIService.java
package com.lc.bangumidemo.MyRetrofit.APIinterface;

import com.lc.bangumidemo.MyRetrofit.ResClass.BookResult;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface APIService {
    @GET("/")
    Call<BookResult> getCall(@Query("xsname") String bookname);
}

File: app/src/main/java/com/lc/bangumidemo/api/APIService.kt
package com.lc.bangumidemo.api

import io.reactivex.Observable
import retrofit2.http.*

const val BASE_URL = "http://api.pingcc.cn/"

interface APIService {

    // 动漫
    @GET(BASE_URL)
    fun getBangumis(@Query("ysname") ysname: String): Observable<SearchResult>

    @GET(BASE_URL)
    fun getBangumiDetail(@Query("ysurl") ysurl: String): Observable<DetailResult>

    // 小说
    @GET(BASE_URL)
    fun getNovels(@Query("xsname") xsname: String): Observable<SearchResult>

    @GET(BASE_URL)
    fun getNovelDetail(@Query("xsurl1") xsurl1: String): Observable<DetailResult>

    @GET(BASE_URL)
    fun getNovelContent(@Query("xsurl2") xsurl2: String): Observable<ContentResult>
}
